packages/icons/src/Icon.js

```diff
@@ -10,29 +10,35 @@ const Icon = ({ name, title, desc, titleId, descId, ...props }) => {
   ariaLabelledBy += desc && descId ? ` ${descId}` : ''
   ariaLabelledBy = ariaLabelledBy ? ariaLabelledBy : undefined
 
-  return (
-    Component && (
-      <Component
-        title={title}
-        desc={desc}
-        titleId={titleId}
-        descId={descId}
-        aria-hidden={!!ariaLabelledBy}
-        aria-labelledby={ariaLabelledBy}
-        {...props}
-      />
+  if (!Component) {
+    if (process.env.NODE_ENV !== 'production')
+      console.trace(`icon ${iconName} does not exist`)
+    return null
+  } else {
+    return (
+      Component && (
+        <Component
+          aria-hidden="true"
+          title={title}
+          desc={desc}
+          titleId={titleId}
+          descId={descId}
+          aria-hidden={!!ariaLabelledBy || true}
+          aria-labelledby={ariaLabelledBy}
+          tabIndex={-1}
+          focusable={false}
```

Comment:
I think this might need to be focusable='false', but if you're seeing it correctly in the DOM, then I'll stand corrected :)

Reply:
I think React suggests to use JavaScript convention like (-1 as a number for tableIdex, which is also camel cased by React, and then these `false` / `true` value)